pyartcd/pyartcd/pipelines/build_microshift.py

```diff
@@ -126,8 +127,9 @@ async def run(self):
                                "rebuild.")
                     self._logger.info(message)
                     await self.slack_say(message)
-                    return
-                nvrs = await self._find_builds()
+                    nvrs = pinned_nvrs.values()
+                else:
+                    nvrs = await self._find_builds()
```

Comment:
A small nit, but nvrs will have two types depending on the if-else, it can be `dict_values` or `list`. Ofc it won't make a difference since I don't see anything that will break, but if in future we try to access it as a list, it will fail since it could also be a `dict_values`.

Reply:
That's a good point! I'll change it